### DISCUSSION TITLE
How about add formula about date

### DISCUSSION THREAD

**Discussion by qjcjerry** [nocodb/nocodb#481] [Features]
How about add formula about date, for example, calulate duration about tow date type columns.

**Comment by YouPray** [comment]
I would like to filter the rows with a date within the next month, for instance

**Comment by pranavxc** [comment]
We also thought about it, in order to make it work we need to figure out how each database handles date comparison. We will add this feature soon.

**Reply by DejanGegic** [reply]
It would be amazing if you could add it just for postgres and/or noco local DB.
